src/stylesheets/VideoListingPage.css

```diff
@@ -0,0 +1,158 @@
+.ff-container {
+  display: grid;
+  grid-template-columns: 2fr 8fr;
+  position: relative;
+  top: 76px;
+}
+
+.ff-container-content {
+  background-color: var(--warning-bg-light);
+  padding: 1rem;
+}
+
+.ff-container-aside-wrap {
+  padding: 1rem;
+  background-color: var(--ff-secondary);
+}
+
+.ff-container-aside {
+  display: grid;
+  grid-template-rows: repeat(5, minmax(45px, 1fr));
+  height: 350px;
+}
+
+.ff-container-aside-chip {
+  padding-inline: 1rem;
+  background-color: var(--ff-secondary);
+}
+
+.ff-aside-chip-icon {
+  color: var(--bg-white);
+  margin: 0.5rem;
+}
+
+.ff-aside-chip-text {
+  color: var(--bg-white);
+  font-size: 1rem;
+  letter-spacing: 1.5px;
+  margin: 0.5rem;
+  font-weight: 400;
+}
+
+.ff-category-text {
+  margin: 0.5rem;
+  padding: 0.5rem;
+  font-size: 0.9rem;
+  font-weight: 500;
+  letter-spacing: 1.5px;
+  border: 1px solid;
+  border-radius: 0.2rem;
+}
+
+.ff-container-video-card {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
+}
+
+.ff-video-card {
+  padding: 0.5rem;
+  margin: 0.5rem;
+  border-radius: 0.2rem;
+  color: var(--warning);
+  background-color: var(--bg-white);
+  box-shadow: 0 1px 3px 0 rgb(0 0 0 / 10%), 0 1px 2px 0 rgb(0 0 0 / 6%);
+  position: relative;
+}
+
+.ff-video-card-content-header {
+  font-weight: 600;
+}
+
+.ff-video-card-content-footer {
+  font-size: 0.8rem;
+  font-weight: 500;
+}
+
+.ff-avatar {
+  width: 2rem;
+  height: 2rem;
+  border-radius: 100%;
+  position: relative;
+  margin-inline: 0.5rem;
+}
+
+.ff-icon-filter {
+  display: block;
+  min-width: 20px;
+  height: 20px;
+  justify-self: flex-start;
+}
+
+.ff-popup-chip {
+  padding: 0.1rem;
+  gap: 0.2rem;
+  font-size: 0.9rem;
+}
+
+.ff-popup {
+  padding: 0.3rem;
+  margin: 0.3rem;
+  position: absolute;
+  bottom: 5px;
+  right: 25px;
+  background-color: var(--bg-white);
+  border-radius: 0.1rem;
+  z-index: 99;
+}
+
+.ff-popup-sort {
+  padding: 0.3rem;
+  margin: 0.3rem;
+  position: absolute;
+  top: 30px;
+  right: 70px;
+  background-color: var(--bg-white);
+  border-radius: 0.1rem;
+  z-index: 99;
+}
+
+.ff-arrowup {
+  position: fixed;
+  z-index: 99999;
```

Comment:
Avoid assigning such high values of z-index to an element. As it might cause issues later on when trying to add new components having similar z-index values such as modals

Reply:
fixed